package.json

```diff
@@ -128,6 +128,7 @@
     "eslint": "^0.15.0",
     "hcat": "0.0.5",
     "jsdoc": "^3.3.0-beta1",
+    "lerna": "https://github.com/kittens/lerna/archive/master.tar.gz",
```

Comment:
@tmcw Can we depend on lerna through npm? 

[babel reference](https://github.com/babel/babel/blob/master/package.json#L24)